### PR TITLE
Add metadata to Irish

### PR DESCRIPTION
**Pull request by daniel0710goldberg**
Add metadata to `irish` dataset.

- [x] Update the information on the dataset's metadata.yaml file.

- [x] Verify the source: [Google Colab notebook](https://colab.research.google.com/drive/1gB7r_CN8LrWG3nOqCS3AXJ7enj_Ssavk?usp=sharing)

### CODE REVIEW

**Comment by daniel0710goldberg** [comment]
This is my first pull request - let me know if anything needs changing or improving.

File: datasets/irish/metadata.yaml [hunk 1]
```diff
@@ -1,24 +1,40 @@
...
     type: discrete
+    description: Prestige score for father's occupation as calculated by Raftery and Hout, 1985
+    code: >
+      '18'=0, '27'=1, '28'=2, '30'=3, '31'=4, '33'=5, '35'=6, '36'=7, '37'=8, '38'=9, '40'=10, '42'=11, '43'=12, 
+      '46'=13, '47'=14, '48'=15, '51'=16, '53'=17, '57'=18, '58'=19, '61'=20, '62'=21, '64'=22, '65'=23, '66'=24,
+      '69'=25, '71'=26, '75'=27, '?'=28
   - name: Type_school
     type: discrete
+    description: Type of schooling
+    code: 'secondary'=1, 'vocational'=2, 'primary_terminal_leaver'=9
```
Comment (trangdata) [inline datasets/irish/metadata.yaml]:
```suggestion
    code: 'secondary'=1, 'vocational'=2, 'primary_terminal_leaver'=0
```

File: datasets/irish/metadata.yaml [hunk 2]
```diff
@@ -1,24 +1,40 @@
...
+      'Senior_cycle_terminal_leaver-secondary_school'=10
   - name: Prestige_score
     type: discrete
+    description: Prestige score for father's occupation as calculated by Raftery and Hout, 1985
+    code: >
+      '18'=0, '27'=1, '28'=2, '30'=3, '31'=4, '33'=5, '35'=6, '36'=7, '37'=8, '38'=9, '40'=10, '42'=11, '43'=12, 
+      '46'=13, '47'=14, '48'=15, '51'=16, '53'=17, '57'=18, '58'=19, '61'=20, '62'=21, '64'=22, '65'=23, '66'=24,
+      '69'=25, '71'=26, '75'=27, '?'=28
   - name: Type_school
     type: discrete
```
Comment (trangdata) [inline datasets/irish/metadata.yaml]:
```suggestion
    type: nominal
```

File: datasets/irish/metadata.yaml [hunk 3]
```diff
@@ -1,24 +1,40 @@
...
-    code: null # optional, coding information, e.g., Control = 0, Case = 1
+    description: Sex of child # optional but recommended, what the feature measures/indicates, unit
+    code: 'male'=1, 'female'=0 # optional, coding information, e.g., Control = 0, Case = 1
     transform: ~ # optional, any transformation performed on the feature, e.g., log scaled
   - name: DVRT
-    type: continuous
+    type: ordinal
+    description: Drumcondra Verbal Reasoning Test score
   - name: Educational_level
     type: discrete
```
Comment (trangdata) [inline datasets/irish/metadata.yaml]: I think, given the automatic coding, we can leave this as nominal as well.

```suggestion
    type: nominal
```

File: datasets/irish/metadata.yaml [hunk 4]
```diff
@@ -1,24 +1,40 @@
...
+    description: attained education level
+    code: >
+      '3rd_level_complete'=0, '3rd_level_incomplete'=1, '?'=2, 'Junior_cycle_incomplete-secondary_school'=3, 
+      'Junior_cycle_incomplete-vocational_school'=4, 'Junior_cycle_terminal_leaver-secondary_school'=5,
+      'Junior_cycle_terminal_leaver-vocational_school'=6, 'Primary_terminal_leaver'=7,
+      'Senior_cycle_incomplete-secondary_school'=8, 'Senior_cycle_incomplete-vocational_school'=9,
+      'Senior_cycle_terminal_leaver-secondary_school'=10
   - name: Prestige_score
     type: discrete
```
Comment (trangdata) [inline datasets/irish/metadata.yaml]:
```suggestion
    type: ordinal
```

Maybe? Was there an order in prestige score calculated by Raftery and Hout, 1985?

**Review by trangdata — COMMENTED** [review]
Nice work @daniel0710goldberg! A few comments and a typo fix.

File: datasets/irish/metadata.yaml (hunk 2 above)
Comment (daniel0710goldberg) [reply datasets/irish/metadata.yaml]: Yeah, I agree

File: datasets/irish/metadata.yaml (hunk 3 above)
Comment (daniel0710goldberg) [reply datasets/irish/metadata.yaml]: Sounds good

File: datasets/irish/metadata.yaml (hunk 4 above)
Comment (daniel0710goldberg) [reply datasets/irish/metadata.yaml]: Yes, higher values correlated to higher prestige. Ordinal makes sense.

File: datasets/irish/metadata.yaml [hunk 5]
```diff
@@ -1,24 +1,46 @@
...
-    description: null # optional but recommended, what the feature measures/indicates, unit
-    code: null # optional, coding information, e.g., Control = 0, Case = 1
-    transform: ~ # optional, any transformation performed on the feature, e.g., log scaled
+    description: Sex of child # optional but recommended, what the feature measures/indicates, unit
+    code: >
+      'male'=1,
+      'female'=0
   - name: DVRT
-    type: continuous
+    type: ordinal
```
Comment (trangdata) [inline datasets/irish/metadata.yaml]: I know these are integers, but given the range, maybe we should consider these as continuous.

**Pull request merged**